discussions/api.py

```diff
@@ -155,7 +284,48 @@ def add_channel(original_search, title, name, public_description, channel_type):
             query=updated_search.to_dict(),
             source_type=PercolateQuery.DISCUSSION_CHANNEL_TYPE,
         )
-        return Channel.objects.create(
+        channel = Channel.objects.create(
             query=percolate_query,
             name=name,
         )
+
+    # Do a one time sync of all matching profiles
+    user_ids = search_for_field(updated_search, 'user_id')
+    from discussions import tasks
+    tasks.add_contributors.delay(channel.name, user_ids)
```

Comment:
This is now erroring because `user_ids` is a `set`, which isn't JSON serializable.

Reply:
Ah right